src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/wwwroot/lib/jquery-validation/.bower.json

```diff
@@ -27,14 +27,15 @@
   "dependencies": {
     "jquery": ">= 1.7.2"
   },
-  "version": "1.14.0",
-  "_release": "1.14.0",
+  "version": "1.17.0",
```

Comment:
I think we need to match these versions in the following file too:
https://github.com/aspnet/templating/blob/release/2.1/src/THIRD-PARTY-NOTICES#L170
cc @Eilon

Reply:
Done